app/gui/src/project-view/components/GraphEditor/GraphNode.vue

```diff
@@ -453,6 +463,10 @@ const { editingComment } = provideComponentButtons(
 )
 
 const showMenuAt = ref<{ x: number; y: number }>()
+onBlur(() => {
+  graph.setNodeHovered(nodeId.value, false)
+  updateNodeHover(undefined)
+})
```

Comment:
Is it this component's blur or window/document blur? I think it should be the latter, but the name does not say it.

Reply:
Renamed to `onWindowBlur`, you are right.